segmentation_labeling_app/transforms/transformations.py

```diff
@@ -226,4 +228,3 @@ def transform_to_mp4(video: np.ndarray, output_path: str,
     for frame in norm_video:
         writer.send(frame)
     writer.close()
-
```

Comment:
If this is the last line I think pep8 says you need a line delimiter at the end of a file

Reply:
I agree, we should all be striving for pep8. FYI, here's the linting output on the master branch for this file. Please start checking this in some way in your code submissions.
```
flake8 segmentation_labeling_app/transforms/transformations.py 
segmentation_labeling_app/transforms/transformations.py:25:80: E501 line too long (80 > 79 characters)
segmentation_labeling_app/transforms/transformations.py:33:80: E501 line too long (83 > 79 characters)
segmentation_labeling_app/transforms/transformations.py:99:80: E501 line too long (80 > 79 characters)
segmentation_labeling_app/transforms/transformations.py:101:80: E501 line too long (83 > 79 characters)
segmentation_labeling_app/transforms/transformations.py:102:80: E501 line too long (80 > 79 characters)
segmentation_labeling_app/transforms/transformations.py:110:80: E501 line too long (82 > 79 characters)
segmentation_labeling_app/transforms/transformations.py:124:80: E501 line too long (81 > 79 characters)
segmentation_labeling_app/transforms/transformations.py:137:80: E501 line too long (82 > 79 characters)
segmentation_labeling_app/transforms/transformations.py:141:80: E501 line too long (82 > 79 characters)
segmentation_labeling_app/transforms/transformations.py:155:80: E501 line too long (80 > 79 characters)
segmentation_labeling_app/transforms/transformations.py:171:80: E501 line too long (88 > 79 characters)
segmentation_labeling_app/transforms/transformations.py:186:80: E501 line too long (81 > 79 characters)
segmentation_labeling_app/transforms/transformations.py:229:1: W391 blank line at end of file
```

Reply:
I think if you try it, you will see that there is a line delimiter at the end of the file, there is not a blank line though. These are different.

Reply:
I agree with pep8 other than line length, I think it makes the code harder to read if there's a hard line at 80 chars but that's a discussion for slack not code comments hah. I see the delimiter that's my mistake.